docs/flocks-settings/default-ignore-list.md

```diff
@@ -227,52 +227,646 @@ These endpoints will set the Ignore Lists for specified Flocks. For Globally def
 
 ## Add IP and Port to Ignore List
 
-<APIDetails :endpoint="$page.frontmatter.endpoints.ignore_ip_ignore_ip_port"/>
+<APIDetails :endpoint="$page.frontmatter.endpoints.ignore_ip_ignore_ip_port">
+
+::::: slot example
+
+:::: tabs :options="{ useUrlFragment: false }"
+
+::: tab "cURL"
+
+``` bash
+curl https://EXAMPLE.canary.tools/api/v1/flock/settings/whitelisting/ip/whitelist_ip_port \
+  -d auth_token=EXAMPLE_AUTH_TOKEN \
+  -d flock_id=EXAMPLE_FLOCK_ID \
+  -d src_ip=EXAMPLE_SOURCE_IP \
+  -d dst_port=EXAMPLE_DESTINATION_PORT
+```
+
+:::
+
+::: tab "Python"
+
+``` python
+import requests
+
+url = 'https://EXAMPLE.canary.tools/api/v1/flock/settings/whitelisting/ip/whitelist_ip_port'
+
+payload = {
+  'auth_token': 'EXAMPLE_AUTH_TOKEN',
+  'flock_id': 'EXAMPLE_FLOCK_ID',
+  'src_ip': 'EXAMPLE_SOURCE_IP',
+  'dst_port': 'EXAMPLE_DESTINATION_PORT'
```

Comment:
This should be an int?

While you are at it, it seems the docs specify this as a `type: false` , please can you change this to `type: int`.